R/otu_table.R

```diff
@@ -21,15 +24,24 @@ read_qiime_otu_table <- function(filepath, commented=TRUE, metadata=TRUE) {
   }
   col_names <- strsplit(header_line, "\t")[[1]]
```

Comment:
If you are using `readr`, you can probably skip all this business with the header line.  The purpose here is to leave the sample names unmodified -- `readr` will do this anyway.

For `readr`, can you make one call to `read_csv`, then skip to the bottom where we split up the data columns from the metadata and OTU ID columns?